cmd/geras/main.go

```diff
@@ -54,11 +55,28 @@ func main() {
 	openTSDBAddress := flag.String("opentsdb-address", "", "host:port")
 	refreshInterval := flag.Duration("metrics-refresh-interval", time.Minute*15,
 		"Time between metric name refreshes. Use negative duration to disable refreshes.")
+	allowedMetricNamesRE := flag.String("metrics-allowed-regexp", ".*", "Regexp of metrics to allow")
```

Comment:
How expensive is `regexp.MatchString` on `".*"`? Just wondering why not the disable on empty as well?

Reply:
2.7 microseconds on my machine. It doesn't seem worth thinking about it for now, given we know there are other things to optimise.

Reply:
ooh, pricey :)